x-pack/plugins/observability/public/pages/slo_edit/components/common/index_field_selector.tsx

```diff
@@ -42,7 +41,7 @@ export function IndexFieldSelector({
     <EuiFlexItem>
       <EuiFormRow label={label} isInvalid={getFieldState(name).invalid}>
         <Controller
-          defaultValue={ALL_VALUE}
+          defaultValue=""
```

Comment:
I would create a `DEFAULT_VALUE = ""` that can be used on the onChange handler as well ` field.onChange(DEFAULT_VALUE);`